app/views/replies/_reply.html.erb

```diff
@@ -42,9 +52,11 @@
 
       <% if user_signed_in? %>
         <% if reply.user.id == current_user.id %>
-          <%= link_to "Delete", {:action => 'destroy', :controller => "replies", :id => @reply.id, :new_thread_id => @new_thread.id}, :method => "delete", data: { confirm: 'Are you sure?' }, :class => 'reply-link edit-post'%>
+          <span id="edit-reply-btn-<%= reply.id %>" class="btn btn-default" reply-id="<%= reply.id %>"><i class="fa fa-pencil"></i></span>
```

Comment:
I used a span tag instead. Is it ok? This is a javascript button that links to nothing. Another way out is using `javascript:void(0)`  in link_to helper.